File: player.rb
 class Player

    def initialize(nama)
        @name = nama
        @blood = 100
        @manna = 40
    end


    def get_name
        @name
    end

    def get_blood
        @blood
    end

    def get_manna
        @manna
    end

    def set_name name
        @name = name
    end

    def set_blood blood
        @blood = blood
    end

    def set_manna manna
        @manna = manna
    end

end

File: player.php
<?php
class Player {
	private $name;
	private $manna;
	private $blood;

	function __construct($name){
		$this->name = $name;
		$this->manna = 40;
		$this->blood = 100;
	}
	function set_name($name) {
      	$this->name = $name;
    }

    function get_name() {
      	return $this->name;
    }

    function set_blood($blood) {
      	$this->blood = $blood;
    }

    function get_blood() {
      	return $this->blood;
    }
    
    function set_manna($manna) {
      	$this->manna = $manna;
    }

    function get_manna() {
      	return $this->manna;
    }
}
?>